### PR TITLE
Give all old forms a created and updated date

### PR DESCRIPTION
**Pull request by 0atman**
Thanks to @thomasiles for suggesting this in #43

### CODE REVIEW

**Review by DavidBiddle — APPROVED** [review]
Looks good to me 🎉

**Comment by 0atman** [comment]
Do merge this before Monday if you would like the migration to happen. Otherwise I'll do so then!

File: spec/db/migration_7_spec.rb
Comment (0atman) [inline spec/db/migration_7_spec.rb:11]: Wait don't merge, I just noticed I don't explicityly create any test cases. Though there SHOULD be stuff in the db to test, I'd want to add some.

**Comment by 0atman** [comment]
oh good my change failed rubocop, please don't fix!

**Review by aliuk2012 — CHANGES_REQUESTED** [review]
Am I missing something, where is the migration 7 file?

**Comment by 0atman** [comment]
It's dB test 7, not migration test 7. Maybe that naming is confusing...

On August 18, 2022, GitHub ***@***.***> wrote:
> @aliuk2012 requested changes on this pull request.
>
> Am I missing something, where is the migration 7 file?
>

—
Reply to this email directly, view it on GitHub
<https://github.com/alphagov/forms-api/pull/57#pullrequestreview-
1076887891>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AAA33MNMOBGIFAR6YLATLFTVZXWI7ANCNFSM562I46JA>.
You are receiving this because you were assigned.Message ID:
***@***.***>

File: Dockerfile
Comment (aliuk2012) [inline Dockerfile:12]: I dont think this should of been part of your changes

File: Gemfile
Comment (aliuk2012) [inline Gemfile:10]: I dont think this should of been part of your changes

File: manifest.yml
Comment (aliuk2012) [inline manifest.yml:13]: I dont think this should of been part of your changes

**Comment by 0atman** [comment]
sorry folks I'm not used to the new merge strategy on git yet, I pulled in some commits from main that I'm to lazy to remove. New PR incoming

**Comment by 0atman** [comment]
https://github.com/alphagov/forms-api/pull/60

**Pull request closed without merging**
